src/components/shared/Carousel.tsx

```diff
@@ -35,6 +35,9 @@ function Carousel(props: CarouselProps): JSX.Element {
   const [slideIdx, setSlideIdx] = useState(0);
   const [child, setChild] = useState(props.children[slideIdx]);
   const [reloadTime, setReloadTime] = useState(Date.now());
+  const [isAutoAdvance, setAutoAdvance] = useState(false);
+  const [isMuted, setMute] = useState(false);
```

Comment:
nit: (and propagate) usually for state variables, you try to do `const [xxx, setXxx] = useState...`
```suggestion
  const [isMuted, setIsMuted] = useState(false);
```